lib/components/inputs/FormCustomSelect.tsx

```diff
@@ -1,49 +1,24 @@
-import { FormInputProps } from "../../types";
-import { Controller, useFormContext } from "react-hook-form";
-
 import { CustomSelect } from "@atomicjolt/atomic-elements";
 import type { CustomSelectProps } from "@atomicjolt/atomic-elements";
+import { Controller } from "react-hook-form";
+import { FormInputProps } from "../../types";
+import { useControllerField } from "../../hooks/useControllerField";
 
-import { splitFormProps } from "../../hooks/useFormInput";
-import { errorMessage } from "../errors";
-
-export type FormCustomSelectProps<T extends object> = FormInputProps<
-  CustomSelectProps<T>,
```

Comment:
Are these no longer needed? Not sure I understand the change to selectedKey

Reply:
I'm not sure I understand what you mean.

The change to `selectedKey` just has to do with the semantics of Atomic Elements 2.0

Using the CustomSelect from Atomic Elements 2.0:
```jsx
<CustomSelect selectedKey="val1">
    <Item key="val1">Item 1</Item>
    <Item key="val2>Item 2</Item>
</CustomSelect>
```

And since this is a wrapper for the elements component, it uses the same props